.github/workflows/tests.yml

```diff
@@ -17,7 +17,21 @@ jobs:
         run: |
           python -m pip install --upgrade pip pipenv
           pipenv install --dev
-      - name: Test
-        run: pipenv run pytest
-      - name: Coveralls
-        run: pipenv run coveralls
\ No newline at end of file
+      - name: Tests
+        run: make test
```

Comment:
You will probably want to run `make coveralls` here so that it will do both the tests and the coveralls report.